index.html

```diff
@@ -0,0 +1,122 @@
+            <ul class="main-nav-ul float-right">
+                <li class="main-nav-i"><i class="fa-brands fa-facebook-f"></i></li>
+                <li class="main-nav-i"><i class="fa-brands fa-twitter"></i></li>
+                <li class="main-nav-i">English</li>
+                <li class="main-nav-i">My Page</li>
+                <li class="main-nav-i">Logout</li>
+            </ul>
+        </div>
+    </nav>
+    <nav class="second-nav fixed-top-nav" id="first-header">
+        <div class="container">
+            <i class="fa-solid fa-bars" id="menu-image"></i>
+            <div class="navigation">
+                <img class="navigation-logo" src="assets/musicFestivalLogo.png" width="65px" alt="Festival">
+                <ul class="second-nav-ul float-right">
+                    <li><a class="second-nav-i" href="about.html">About</a></li><br>
+                    <li><a class="second-nav-i" href="#program">Program</a></li><br>
+                    <li><a class="second-nav-i" href="#join">Join</a></li><br>
+                    <li><a class="second-nav-i" href="#sponsor">Sponsor</a></li><br>
+                    <li><a class="second-nav-i" href="#updates">News</a></li><br>
+                    <li><a href="#" class="mt-3 p-2" id="cc-btn">CC Campaign</a></li><br>
+                </ul>
+            </div>
+        </div>
+    </nav>
+    <nav class="mobile-nav">
+        <span class="cross">x</span>
+        <ul class="mobile-nav-ul">
+            <li><a class="mobile-nav-i" href="about.html">About</a></li><br>
+            <li><a class="mobile-nav-i" href="#program">Program</a></li><br>
+            <li><a class="mobile-nav-i" href="#join">Join</a></li><br>
+            <li><a class="mobile-nav-i" href="#sponsor">Sponsor</a></li><br>
+            <li><a class="mobile-nav-i" href="#updates">News</a></li><br>
+            <li><a href="#" class="mt-3 p-2" id="cc-btn">CC Campaign</a></li><br>
+        </ul>
+    </nav>
+    <main class="top-section">
+        <section class="top-bg">
+            <section class="hero container"><br><br><br><br>
+                <h2 class="salute">"Hello! Sharing World"</h2>
+                <h1 class="hero-first-title">Kings of Music<br>Festival</h1><br>
+            </section>
+            <section class="text-message">
+                <div class="container">
+                    <p class="message-p p-3 border border-white">Grab your friends, grab your dancing shoes and let's go! Let's make this festival an unforgettable experience!</p>
+                    <h2 class="message-time">📆23 June(Fri) to 25 June(Sun) 2023</h2>
+                    <p>📍108 Holland Drive, Malibu.</p>
+                </div>
+            </section><br><br><br>
+        </section>
+
+        <section class="program" id="program">
+            <br>
+            <h4 class="text-center section-title-white">Main Program</h4>
+            <hr class="line-name">
+            <div class="program-container container">
```

Comment:
Good job implementing the `Main Program` section. To match better with the Behance design 

- [ ] Kindly reduce the gap between each card

- [ ] Change the font-family text in this section for `Lato`

- [ ] Reduce the size of your description text

- [ ] And lastly, as you can see the card container is centered in the middle of the section. You could reduce the space between the cards and the section to center it in the middle, maybe adjusting the size of the container or using margins.

![image](https://github.com/Tarto-4/First-Capstone/assets/93630700/67dc0126-249d-4b43-96d7-946356db1536)

![image](https://github.com/Tarto-4/First-Capstone/assets/93630700/ac6b767b-3ce1-4252-9dc4-851406d3975d)